### ISSUE TITLE
[Firefox -72] - addonlists_show_buttons_instead_of_menu_popup_fx68.css - not working

### ISSUE THREAD

**Issue by Acid-Crash** [Aris-t2/CustomCSSforFx#246]
1. Describe the issue
_addonlists_show_buttons_instead_of_menu_popup_fx68.css_ stopped working. 3-dot menu is gone but no buttons are visible (apart from one on the top)

2. What should have happened instead?
The module should be working

3. Steps to reproduce the issue?
- Enable _addonlists_show_buttons_instead_of_menu_popup_fx68.css_
- go to _about:addons_

4. SCREENSHOTS with/without this projects files (drag & drop images into this post)
![изображение](https://user-images.githubusercontent.com/32600318/68071692-86780100-fd85-11e9-8a25-3b566c7951a0.png)

5. System information
- OS & OS version: Win10x64
- OS theme name: default
- DPI / HiDPI resolution: 100
- Firefox version: 72
- Firefox theme / lw-theme: default
- This projects settings the issue occurs with: 

<details>

```css

/* Firefox Quantum userContent.css tweaks ********************************************************/
/* code mostly taken from 'Classic Theme Restorer' & 'Classic Toolbar Buttons' add-ons **********/
/* Github: https://github.com/aris-t2/customcssforfx ********************************************/
/************************************************************************************************/

/*************************************************************************************************

[!] Firefox 68+ (look for '_fx68')
  - new 'alternative appearance' option
  - new 'alternative appearance' option (AERO background color)
  - new 'restore version number' option for active add-ons
  - new 'compact appearance' option
  - new 'more compact appearance' option
  - new 'compact categories' option
  - new 'replace menu popup with buttons' option
  - new 'replace button labels with icons' option
  - new 'colorized icon' option for 'replace button labels with icons' option
  - new 'show buttons on hover only' option
  - new 'hide manage/expand button' option
  - new 'hide report button' option
  
  - old add-ons manager > show 'version number for add-ons' does not work with the html based add-on manager
  - old add-ons manager > show 'last update date' does not work with the html based add-on manager
  - old add-ons manager > alternative appearance does not work with the html based add-on manager
  - old add-ons manager > compact appearance does not work with the html based add-on manager
  - most old 'other' add-ons manager settings do not work with the html based add-on manager
  --> use 'about:config > extensions.htmlaboutaddons.enabled > false' temporary workaround for the old settings
  
[!] Firefox 69+
  - 'about:config > toolkit.legacyUserProfileCustomizations.stylesheets > true' (or custom styles will not be loaded)
  
[!] Firefox 70+
  - 'about:config >  extensions.htmlaboutaddons.enabled > false' workaround does not work anymore
  
  
*************************************************************************************************/

/*************************************************************************************************

  README

  Finding profile folder: address bar > about:support > Profile Folder > Open Folder
  CSS files and sub-folders belong into \PROFILEFOLDER\chrome\ directory.
  \chrome\config\
  \chrome\css\
  \chrome\image\
  \chrome\userChrome.css
  \chrome\userContent.css

  ENABLING options: remove the combination of / and * before '@import'
  DISABLING options: add the combination of / and * before '@import'

  FIREFOX BUGS (nothing CSS code can do about):
  - some '@import' calls might not work properly on macOS and Linux
  - some '@import' calls might not work properly, if multi-process mode (e10s) is disabled
  - in both cases CSS code has to be added to userContent.css file manually

  Some CSS tweaks won't work, if '@namespace' references are used inside .css files!

  All options have significant categories and names.

  GENERAL SUPPORT / DISCUSSION THREAD: github.com/Aris-t2/CustomCSSforFx/issues/211

*************************************************************************************************/


/************************************************************************************************/
/* about:addons (settings split into different areas for Firefox 60-67, 68+ or 60-70+) **********/
/************************************************************************************************/

/* FIREFOX 68+ #################################################################################*/
/* addons page appearance - [only use one at a time] (incl. version number & buttons) ***********/
/* @import "./css/aboutaddons/addons_manager_alternative_appearance_fx68.css"; /**/
/* @import "./css/aboutaddons/addons_manager_alternative_appearance_aero_fx68.css"; /**/    /* <-- AERO colors */

/* version number for active add-ons [Fx 68+] ***************************************************/
@import "./css/aboutaddons/addonlists_show_addon_version_number_fx68.css"; /**/

/* compact item lists - [only use one at a time] ************************************************/
/* @import "./css/aboutaddons/addonlists_compact_fx68.css"; /**/ /* best with alternative_appearance_fx68 */
/* @import "./css/aboutaddons/addonlists_compact_more_compact_fx68.css"; /**/ /* best with alternative_appearance_fx68 */

/* button settings (requires addons_manager_alternative_appearance_fx68) ************************/
@import "./css/aboutaddons/addonlists_show_buttons_instead_of_menu_popup_fx68.css"; /**/
@import "./css/aboutaddons/addonlists_hide_report_button_fx68.css"; /**/
@import "./css/aboutaddons/addonlists_hide_manage_button_fx68.css"; /**/
/* @import "./css/aboutaddons/addonlists_replace_button_labels_with_icons_fx68.css"; /**/
/* @import "./css/aboutaddons/addonlists_colorized_button_icons_fx68.css"; /**/ /* requires replace_button_labels_with_icons option */
/* @import "./css/aboutaddons/addonlists_only_show_buttons_on_hover_fx68.css"; /**/

/* addons page appearance extra [requires one 'alternative appearance option' to be active] *****/
/* @import "./css/aboutaddons/addons_manager_alternative_appearance_compact_categories_fx68.css"; /**/

/* other settings *******************************************************************************/
/* @import "./css/aboutaddons/addonlists_hide_recommended_icon_fx68.css"; /**/
/* @import "./css/aboutaddons/addonlists_hide_allowed_in_private_windows_icon_fx68.css"; /**/


/* FIREFOX 60-67 ###############################################################################*/
/* NOTE: 'Version number for add-ons' will stop working soon when Mozilla drops XBL support. *****
         See https://bugzilla.mozilla.org/show_bug.cgi?id=1397874 *******************************/

/* version number for add-ons [Fx 60-67] - [only use one at a time] *****************************/
/* [!] version number's xml file now restores add-on 'description' on Firefox 64+ ***************/
/* @import "./css/aboutaddons/addonlists_show_addon_version_number.css"; /**/
/* @import "./css/aboutaddons/addonlists_show_addon_version_number_hide_description.css";

/* show 'last updated' date for add-ons [Fx 60-67] - [only use one at a time] *******************/
/* @import "./css/aboutaddons/addonlists_show_addon_date_last_updated.css"; /**/
/* @import "./css/aboutaddons/addonlists_show_addon_date_last_updated_start_position.css"; /**/

/* addons page appearance [Fx 60-67] - [only use one at a time] *********************************/
/* @import "./css/aboutaddons/addons_manager_alternative_appearance.css"; /**/
/* @import "./css/aboutaddons/addons_manager_alternative_appearance_aero.css"; /**/ /* <-- AERO colors */
/* @import "./css/aboutaddons/addons_manager_alternative_appearance_v2.css"; /**/
/* @import "./css/aboutaddons/addons_manager_alternative_appearance_v2_aero.css"; /**/  /* <-- AERO colors */

/* addons page appearance extra [requires one 'alternative appearance option' to be active] *****/
/* @import "./css/aboutaddons/addons_manager_alternative_appearance_compact_categories.css"; /**/

/* compact item lists [Fx 60-67] - [only use one at a time] *************************************/
/* @import "./css/aboutaddons/addonlists_compact.css"; /**/
/* @import "./css/aboutaddons/addonlists_compact_more_compact.css"; /**/

/* other settings [Fx 60-67] ********************************************************************/
/* @import "./css/aboutaddons/addonlists_disabled_addons_colorized_icons.css"; /**/ 
/* @import "./css/aboutaddons/addonlists_disabled_tag_hidden.css"; /**/
/* @import "./css/aboutaddons/addonlists_only_show_buttons_on_hover.css"; /**/
/* @import "./css/aboutaddons/addonlists_replace_button_labels_with_icons.css"; /**/    
/* @import "./css/aboutaddons/addonlists_move_button_to_start.css"; /**/
/* @import "./css/aboutaddons/addonlists_green_dot_hidden.css"; /**/    
/* @import "./css/aboutaddons/details_page_alternative_content_order.css"; /**/ 
/* @import "./css/aboutaddons/details_page_switch_banner_label_position_for_themes.css"; /**/
/* @import "./css/aboutaddons/search_category_always_visible.css"; /**/ /* not for recent Fx versions */


/* FIREFOX 60-70+ ##############################################################################*/
@import "./css/aboutaddons/addonlists_private_browsing_notice_hidden.css"; /**/
@import "./css/aboutaddons/recentupdates_category_always_visible.css"; /**/
@import "./css/aboutaddons/recentupdates_category_last_category.css"; /**/
/* @import "./css/aboutaddons/availableupdates_category_always_visible.css"; /**/
@import "./css/aboutaddons/availableupdates_category_last_category.css"; /**/


/************************************************************************************************/
/* about:preferences ****************************************************************************/
/************************************************************************************************/

/* preferences page appearance - [only use one at a time] ***************************************/
/* @import "./css/aboutpreferences/preferences_alternative_appearance.css"; /**/
/* @import "./css/aboutpreferences/preferences_alternative_appearance_aero.css"; /**/   /* <-- AERO colors */
/* @import "./css/aboutpreferences/preferences_alternative_appearance_v2.css"; /**/


/************************************************************************************************/
/* about:newtab / about:home ********************************************************************/
/************************************************************************************************/

/* a dark appearance for inbuilt 'Dark' theme is present by default in Firefox 61+ **************/
/* @import "./css/aboutnewtab/dark_appearance.css"; /**/
/* alternative background color for about:newtab/home pages 'Dark' theme present in Firefox 61+ */
/* @import "./css/aboutnewtab/dark_appearance_alternative_background.css"; /**/


/************************************************************************************************/
/* about:config (Firefox 67+) *******************************************************************/
/************************************************************************************************/

/* @import "./css/aboutconfig/aboutconfig_compact_appearance.css"; /**/


/************************************************************************************************/
/* WEB CONTENT **********************************************************************************/
/************************************************************************************************/

/* top level image on white background - [only use one at a time] *******************************/
/* [!] this settings can sometimes cause glitches ***********************************************/
/* @import "./css/webcontent/toplevel_image_classic_v1.css"; /**/
/* @import "./css/webcontent/toplevel_image_classic_v2.css"; /**/

/* Screenshots - internal browser add-on ********************************************************/
/* @import "./css/webcontent/screenshots_alternative_save_ui.css"; /**/

/* CUSTOM SCROLLBARS VARIABLES (Firefox 63+ only) - EXPERIMENTAL ********************************/
/* [!] Might not work on every page! This issue is for Mozilla to solve. ************************/
/* [!] not compatible with custom scrollbar JavaScipt userChrome scripts ************************/
/* [!] custom size values only compatible with 'web content' scrollbars at the moment ***********/
/* [!] more info inside 'custom_scrollbar_appearance.css' file **********************************/
/* @import "./config/custom_scrollbar_appearance.css"; /**/     /* <--- EXPERIMENTAL */

/* Autofill forms (Firefox 67+ only) - disable CSS3 filter background ***************************/
/* @import "./css/webcontent/autofill_forms_background_white.css"; /**/ 


/************************************************************************************************/
/************************************************************************************************/
/* Create a new file "my_userContent.css" and add own/custom code to it. ************************/
@import "./my_userContent.css"; /**/
/************************************************************************************************/
/************************************************************************************************/
/************************************************************************************************/

```

</details>

**Comment by Aris-t2** [comment]
Devs changed add-on manager code again. This starts to get boring. Don't they have anything better to do? ;-)

**Comment by Acid-Crash** [comment]
True((
Unfortunatelly this addont is one of the mustHave's at least for me))

**Comment by Pizzapops** [comment]
Change for the sake of change and job security! My addons manager columns have been neutered.

**Comment by Aris-t2** [comment]
I found a workaround for the new changes. Next update will offer "new" options.

**Comment by Aris-t2** [comment]
[--> 2.8.2](https://github.com/Aris-t2/CustomCSSforFx/releases)

**Comment by hopalongrock** [comment]
Fx72 (latest nightly): There is no drop-down menu next to gear icon, only "search addons.mozilla.o" part is visible, and there are no buttons on Plugins. The buttons appear on extensions and themes.

![image](https://user-images.githubusercontent.com/6840865/68088287-daeeaf80-fe5d-11e9-9a4d-36a0fcf2898e.png)

**Comment by Aris-t2** [comment]
I replaced the the 2.8.2 file with a fixed version. Changes happened inside `addonlists_show_buttons_instead_of_menu_popup_fx72.css`.

**Comment by hopalongrock** [comment]
Plugins are now OK.
The first line is still different:
FX72:
![image](https://user-images.githubusercontent.com/6840865/68089270-63258280-fe67-11e9-9cb4-2f9f7927009d.png)

Fx70:
![image](https://user-images.githubusercontent.com/6840865/68089296-9ec04c80-fe67-11e9-8c70-824b74d03037.png)

**Comment by Pizzapops** [comment]
@Aris-t2 My problem with _addonlists_show_buttons_instead_of_menu_popup_fx72.css_is the the buttons do not replace the menu popup in the same location. The buttons end up in the far right of the addon page instead of the right side of the addon-card. Is the menu popup no longer contained within each addon-card?
![2019-11-03_E](https://user-images.githubusercontent.com/33039961/68090802-c69aba80-fe3d-11e9-9269-ced82e78cf18.jpg)
Is this something Mozilla Devs did on purpose or a possible bug?

**Comment by Acid-Crash** [comment]
Same here.
Not sure if this is related to the fact that I do not use _addons_manager_alternative_appearance_fx68.css_ (as mentioned in the userContent)
On release and prior to FF72 this was also disabled and buttons_instead_of_menu_popup did work fine...

**Comment by Aris-t2** [comment]
@hopalongrock 
There is always room for improvement, but not everything will be 100% the same it was before.

@Pizzapops @Acid-Crash 
I found the reason for this glitch.

-----

[Get custom_css_for_fx_v2.8.2v3.zip](https://github.com/Aris-t2/CustomCSSforFx/releases/tag/2.8.2)

**Comment by Pizzapops** [comment]
@Aris-t2 You must bed burning the midnight oil! Thanks for the update. 
Can I get a clue for moving the buttons up to the same line the extension name.
![2019-11-03_180723](https://user-images.githubusercontent.com/33039961/68094400-9830d580-fe65-11e9-9c93-cf767b92c428.jpg)

**Comment by Aris-t2** [comment]
Who needs sleep? ;-)

Code should look like this.
```css
addon-list .more-options-menu :-moz-any(addon-options,plugin-options) panel-list {
  margin-top: -40px !important;
}
```

**Comment by Acid-Crash** [comment]
Looks like the top popUp gets an update.
![image](https://user-images.githubusercontent.com/32600318/68126013-83af1480-ff1b-11e9-9180-cffaf4fa01ac.png)
The mentioned module affects it as well, not sore if this is a good thing...
![image](https://user-images.githubusercontent.com/32600318/68126113-bb1dc100-ff1b-11e9-8a65-accedb838db5.png)

**Comment by Aris-t2** [comment]
Unfortunately "buttons" and and top menu use same list node internally.

**Comment by Acid-Crash** [comment]
Ok, in this case guess I need to choose the least among the two evils...

**Comment by Aris-t2** [comment]
[v2.8.3 improves the box again](https://github.com/Aris-t2/CustomCSSforFx/releases/tag/2.8.3)

**Comment by Pizzapops** [comment]
@Aris-t2 
I finally got the buttons aligned with the addon name. I had to change line #43 **margin-top: -25px;** to **top:-25px !important;**  I also added **right: 4px !important;** to keep the buttons from hugging the right side.
I am still trying to shrink the card-contents by about 24px. It seems like it or the more-options-menu are not collapsing like they should. Today's attempts have not been successful.
![2019-A](https://user-images.githubusercontent.com/33039961/68245197-216a1880-ffdc-11e9-9554-a01f417bdcaf.jpg)
